docs/extended_api/synchronization_primitives/atomic_ref.md

```diff
@@ -0,0 +1,101 @@
+---
+grand_parent: Extended API
+parent: Synchronization Primitives
+nav_order: 1
+---
+
+# `cuda::atomic_ref`
+
+Defined in header `<cuda/atomic>`:
+
+```cuda
+template <typename T, cuda::thread_scope Scope = cuda::thread_scope_system>
+class cuda::atomic_ref;
+```
+
+The class template `cuda::atomic_ref` is an extended form of [`cuda::std::atomic_ref`]
+  that takes an additional [`cuda::thread_scope`] argument, defaulted to
+  `cuda::std::thread_scope_system`.
+It has the same interface and semantics as [`cuda::std::atomic_ref`], with the
+  following additional operations.
+This class additionally deviates from the standard by being backported to C++11.
+
+## Limitations
+
+`cuda::atomic_ref<T>` and `cuda::std::atomic_ref<T>` may only be instantiated with types sized between 4 and 8 bytes.
```

Comment:
Is this restriction on `cuda::std::atomic_ref<T>` only described here in this extended API section on `cuda::atomic_ref<T>`?

Reply:
Good point. We probably want to call out the limitation on `cuda::std::atomic_ref<T>` on the "Standard API" section as well.

cc @griwes